### PR TITLE
Create docker-image.yml

### PR DESCRIPTION
**Pull request by jakecomp**
# Add initial docker-compose actions for CI/CD  

This PR contains 2 simple checks for starting up the docker-compose file, and for running all tests on our repo.

### CODE REVIEW

**Comment by jakecomp** [comment]
Okay ... it's been tedious but I finally got both checks working properly

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
```
Comment (jberryman566) [inline .github/workflows/go.yml]: Small detail but, is it possible to name the two checks something more meaningful for devs to understand what is happening. Ex step 1: Compose Setup, step 2: Build & Test?

Comment (jakecomp) [reply .github/workflows/go.yml]: Can do!

**Review by jberryman566 — APPROVED** [review]
LGTM! Ship It!

**Pull request merged**